images/multus-cni/README.md

```diff
@@ -0,0 +1,28 @@
+<!--monopod:start-->
+# multus-cni
+| | |
+| - | - |
+| **OCI Reference** | `cgr.dev/chainguard/multus-cni` |
+
+
+* [View Image in Chainguard Academy](https://edu.chainguard.dev/chainguard/chainguard-images/reference/multus-cni/overview/)
+* [View Image Catalog](https://console.enforce.dev/images/catalog) for a full list of available tags.
+* [Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
+
+---
+<!--monopod:end-->
+
+<!--overview:start-->
+A CNI meta-plugin for multi-homed pods in Kubernetes
+<!--overview:end-->
+
+<!--getting:start-->
+## Download this Image
+The image is available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/multus-cni:latest
+```
+<!--getting:end-->
+
+<!--body:start--><!--body:end-->
```

Comment:
We need some documentation with examples on how to use the image, i.e helm chart, docker etc